### PR TITLE
Updates to demo

### PR DESCRIPTION
**Pull request by MattiasGees**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ Once NGINX Plus is deployed you can access the dashboard on port `http://<hostna
 Next we're installing cert-manager. cert-manager will allow you to integrate the Venafi platforms with Kubernetes to issue TLS certificates and provide identity management for applications across your cluster
 To install cert-manager run:
 ```console
-$ kubectl apply -f "https://github.com/jetstack/cert-manager/releases/download/v0.14.1/cert-manager.yaml"
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.3/cert-manager.yaml
```
Comment (meyskens) [inline README.md:162]: --validate=false should no longer be needed, our docs do this because people are bad at updating kubectl

**Comment by meyskens** [comment]
@MattiasGees one nit then lgtm

File: README.md (hunk 1 above)
Comment (MattiasGees) [reply README.md:162]: Should this also be removed from the cert-manager docs (as this is a copy :) )?

Comment (meyskens) [reply README.md:162]: >  people are bad at updating kubectl

maybe not yet even when you look at people's kubectl versions they are quite behind.... but here we do install it ourselves

File: workload.yaml [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
   name: workload-certificate
   namespace: default
 spec:
   secretName: workload-tls
+  subject:
+    organizations:
+      - Example Organization
   dnsNames:
     - workload.demo.example.com
   issuerRef:
```
Comment (wallrj) [inline workload.yaml:14]: nit: There's also a much newer nginx image that we could use in the deployment below: 1.19.3

File: README.md [hunk 3]
```diff
@@ -580,14 +579,15 @@ This can be found in your CloudShare environment under "Connection Details", the
...
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: hello-world-ingress
   namespace: default
   annotations:
     kubernetes.io/ingress.class: "nginx"
     cert-manager.io/issuer: "venafi-tpp-issuer"
+    cert-manager.io/common-name: "<place external hostname here>"
```
Comment (wallrj) [inline README.md:590]: Also need to change the service spec below:

```
kubectl apply -f hello-world.yaml
error: error validating "hello-world.yaml": error validating data: [ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "serviceName" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "servicePort" in io.k8s.api.networking.v1.IngressBackend]; if you choose to ignore these errors, turn validation off with --validate=false
```

File: hello-world.yaml [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
 metadata:
   name: hello-world-ingress
   namespace: default
   annotations:
     kubernetes.io/ingress.class: "nginx"
     cert-manager.io/issuer: "venafi-tpp-issuer"
+    cert-manager.io/common-name: "<place external hostname here>"
 spec:
   tls:
     - hosts:
```
Comment (wallrj) [inline hello-world.yaml:13]: Need to change the service spec below:

```
kubectl apply -f hello-world.yaml
error: error validating "hello-world.yaml": error validating data: [ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "serviceName" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "servicePort" in io.k8s.api.networking.v1.IngressBackend]; if you choose to ignore these errors, turn validation off with --validate=false
```

Comment (wallrj) [reply hello-world.yaml:13]:
```
diff --git a/hello-world.yaml b/hello-world.yaml
index 7f3ae44..edc215d 100644
--- a/hello-world.yaml
+++ b/hello-world.yaml
@@ -7,20 +7,23 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: "nginx"
     cert-manager.io/issuer: "venafi-tpp-issuer"
-    cert-manager.io/common-name: "<place external hostname here>"
+    cert-manager.io/common-name: "uvo146orxer22akg8cm.vm.cld.sr"
 spec:
   tls:
     - hosts:
-        - <place external hostname here>
+        - uvo146orxer22akg8cm.vm.cld.sr
       secretName: venafi-demo-tls
   rules:
-    - host: <place external hostname here>
+    - host: uvo146orxer22akg8cm.vm.cld.sr
       http:
         paths:
           - path: /
+            pathType: Prefix
             backend:
-              serviceName: hello-world-service
-              servicePort: 80
+              service:
+                name: hello-world-service
+                port: 
+                  number: 80
 ---
 apiVersion: apps/v1
 kind: Deployment
```

**Review by wallrj — CHANGES_REQUESTED** [review]
Ah, I spotted a few other things, but I'll create a follow up PR for those.

File: README.md (hunk 1 above)
Comment (wallrj) [reply README.md:162]: Done.

**Pull request merged**